### ISSUE TITLE
Test failures with Python 3.7

### ISSUE THREAD

**Issue by QuLogic** [ionelmc/python-tblib#36]
We are rebuilding everything against Python 3.7 in Fedora, and it looks like [tblib is failing there](https://kojipkgs.fedoraproject.org//work/tasks/3757/27753757/build.log):
```
=================================== FAILURES ===================================
_____________________________ [doctest] README.rst _____________________________
155     ...
156     >>> len(s3) > 1
157     True
158 
159 Unpickling
160 ~~~~~~~~~~
161 
162 ::
163 
164     >>> pickle.loads(s1)
Expected:
    (<...Exception'>, Exception('fail',), <traceback object at ...>)
Got:
    (<class 'Exception'>, Exception('fail'), <traceback object at 0x7f89feecf748>)
/builddir/build/BUILD/tblib-1.3.2/README.rst:164: DocTestFailure
```

It looks like maybe the comma in `Exception('fail',)` is removed now.

**Comment by QuLogic** [comment]
I've applied [this patch](https://src.fedoraproject.org/rpms/python-tblib/blob/master/f/0001-Patch-tests-against-Python-3.7.patch) to get it to pass for the time being.

**Comment by QuLogic** [comment]
Ping? We're starting to look at 3.8 already.

**Comment by ionelmc** [comment]
@QuLogic will kick some CI today to see how it goes.

**Comment by ionelmc** [comment]
Patch included in d4886fb.

**Comment by ionelmc** [comment]
Released `v1.4.0`.
